toadr3/serializer.py

```diff
@@ -0,0 +1,252 @@
+import datetime
+import json
+from typing import Any
+
+from .event import Event
+from .event_payload_descriptor import EventPayloadDescriptor
+from .interval import Interval
+from .interval_period import IntervalPeriod
+from .iso_date import create_iso8601_duration
+from .report import Report, ReportData
+from .report_descriptor import ReportDescriptor
+from .report_payload_descriptor import ReportPayloadDescriptor
+
+
+def _serialize_values_map(values_map: dict[str, list[str, Any]]) -> list[dict[str, Any]]:
+    """Serialize a values map to a JSON compatible object."""
+    result = []
+    for key, values in values_map.items():
+        result.append({"type": key, "values": values})
```

Comment:
If values is Any, you might want to call the ToadrJSONEncoder.default method on each of them?